### PR TITLE
updated example

### PR DESCRIPTION
**Pull request by johndpope**

### CODE REVIEW

File: Example/DetailViewController.swift [hunk 1]
```diff
@@ -72,9 +72,12 @@ class DetailViewController: UITableViewController {
             let end = CACurrentMediaTime()
             self.elapsedTime = end - start
 
-            for (field, value) in response!.allHeaderFields {
-                self.headers["\(field)"] = "\(value)"
+            if let _ = response{
```
Comment (kylef) [inline Example/DetailViewController.swift]: This should be `if let response = response`, and that way you don't have to explicitly unwrap the optional below `response!`.

**Comment by johndpope** [comment]
ok  - the commit was more about fixing the critical crash and getting demo working than reducing the exclamation marks. I've seen this pattern - if let response = response but I think there's a push to abbreviate it. thanks

**Comment by cnoon** [comment]
Thank you for your PR @johndpope. I merged a modified form of your PR in 2bffaf1b giving you attribution. Thanks again!

**Comment by johndpope** [comment]
fyi - this pull request added additional NSAppTransportSecurity to plist fixing the bug with example not working. The example is still not working. 

<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
	</dict>

**Comment by cnoon** [comment]
It works fine for me. I've migrated the example over to use all HTTPS calls which should play nice with ATS. Could you possibly provide more info and open a new PR if necessary? Could you check against the Alamofire 2.0.0-beta.1 release?

**Pull request closed without merging**